src/app/components/developer/developer.html

```diff
@@ -204,7 +204,7 @@ <h2 class="panel-title">Editing - {{ $ctrl.developer.name }}</h2>
       </div>
     </span>
 
-    <span feature-flag="ui-bump-major">
+    <div feature-flag="ui-bump-major">
```

Comment:
I don't see a corresponding change to the closing /span. I don't think this change is needed